src/redux/wallets/actions.ts

```diff
@@ -180,8 +183,8 @@ export const sendTransferTransaction = (
         const options = {
             nonce,
             chainId,
-            gasPrice: FEE[account.blockchain].gasPrice,
-            gasLimit: FEE[account.blockchain].gasLimit
+            gasPrice,
```

Comment:
for now get `gasPrice` and `gasLimit` directly from `feeOptions`. In near future we will change the interface for a transaction, fo have fee options grouped in an object